projects.go

```diff
@@ -464,59 +465,82 @@ func (s *ProjectsService) GetProjectEvents(pid interface{}, opt *GetProjectEvent
 	return p, resp, err
 }
 
+// ProjectContainerExpirationPolicy represents the container expiration attributes for a project.
+type ProjectContainerExpirationPolicy struct {
+	Cadence         string    `json:"cadence"`
+	KeepN           uint      `json:"keep_n"`
+	OlderThan       string    `json:"older_than"`
+	NameRegex       string    `json:"name_regex"` // NameRegex will be replaced by NameRegexDelete but now calls without setting this field are not accepted.
+	NameRegexDelete string    `json:"name_regex_delete"`
+	NameRegexKeep   string    `json:"name_regex_keep"`
+	Enabled         bool      `json:"enabled"`
+	NextRunAt       time.Time `json:"next_run_at"`
+}
+
+// ProjectContainerExpirationPolicyAttributes represents the container expiration attributes for a project.
+type ProjectContainerExpirationPolicyAttributes struct {
+	Cadence         string `json:"cadence"`
+	KeepN           uint   `json:"keep_n"`
+	OlderThan       string `json:"older_than"`
+	NameRegex       string `json:"name_regex"` // NameRegex will be replaced by NameRegexDelete but now calls without setting this field are not accepted.
+	NameRegexDelete string `json:"name_regex_delete"`
+	NameRegexKeep   string `json:"name_regex_keep"`
+	Enabled         bool   `json:"enabled"`
+}
```

Comment:
I think these fields all need to be pointers and have `omitempty` added to the tags. Additionally please also add the `url` tags to make sure this new code is consistent.

Reply:
Changed as requested